File: 0501_0600/0550/0550.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n = int(f.readline())
    q.write(f"{12 if n % 400 == 0 or (n % 100 and n % 4 == 0) else 13}/09/{n:04d}")

File: 0501_0600/0550/0550.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n;

	fscanf(f, "%d", &n);
	fprintf(q, "%d/09/%04d", !(n % 400) || (n % 100 && !(n % 4)) ? 12 : 13, n);
	return 0;
}
